package.json

```diff
@@ -4,6 +4,7 @@
   "description": "A simple, fast, robust job/task queue, backed by Redis.",
   "main": "index.js",
   "dependencies": {
+    "p-finally": "^1.0.0",
     "promise-callbacks": "^3.0.0",
```

Comment:
This change needs at least v3.4.0 to work as intended- https://github.com/bee-queue/bee-queue/pull/215/commits/0cfba99956b700c420852e9f342528ab8b8b1da4

Reply:
9c456e1a5dc619f38d67b3ab3d92cce3b0d162bc